### ISSUE TITLE
Unexpected token { in JSON

### ISSUE THREAD

**Issue by nicinabox** [peerigon/updtr#59]
Just updated to 1.0.0 and got this error when running (Node v7.5.0):

```
❯ updtr
Looking for outdated modules... ⠲ undefined:2
{"type":"finished","data":978}
^

SyntaxError: Unexpected token { in JSON at position 922
    at JSON.parse (<anonymous>)
    at /usr/local/lib/node_modules/updtr/lib/run.js:122:25
    at ChildProcess.exithandler (child_process.js:202:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:192:7)
    at maybeClose (internal/child_process.js:890:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at Pipe._handle.close [as _onclose] (net.js:501:12)
```

**Comment by jhnns** [comment]
Do you have a `yarn.lock` file in your project dir?

**Comment by nicinabox** [comment]
I do

**Comment by a7madgamal** [comment]
```Running updtr with custom configuration:

● use: yarn

Looking for outdated modules...
> yarn outdated --json --flat 
⠴

 ERROR  Error when trying to parse stdout from command 'yarn outdated --json --flat': Unexpected token { in JSON at position 222
    at Object.parse (native)
    at parse$1 (/Users/miamed/.config/yarn/global/node_modules/updtr/dist/index.js:1424:17)
    at Object.outdated (/Users/miamed/.config/yarn/global/node_modules/updtr/dist/index.js:1440:31)
    at getUpdateTasksFromStdout (/Users/miamed/.config/yarn/global/node_modules/updtr/dist/index.js:684:5)
    at _callee$ (/Users/miamed/.config/yarn/global/node_modules/updtr/dist/index.js:720:42)
    at tryCatch (/Users/miamed/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:65:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/miamed/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (/Users/miamed/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/Users/miamed/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /Users/miamed/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:30:13
```

**Comment by brekk** [comment]
If anyone else is delayed by this, you can use https://github.com/fritx/jayin as a quick (but blind) workaround.

```bash
cat package.json |\
  jayin "Object.keys(x.dependencies).join(' ')" |\
  xargs echo |\
  xargs yarn add -S
```

Or for devDependencies:

```bash
cat package.json |\
  jayin "Object.keys(x.devDependencies).join(' ')" |\
  xargs echo |\
  xargs yarn add -D
```

YMMV.

**Comment by GantMan** [comment]
I'm also having this issue.

**Comment by brekk** [comment]
Recently learned about [`yarn upgrade-interactive`](https://yarnpkg.com/lang/en/docs/cli/upgrade-interactive/), which ought to solve your problems well (if you're using yarn)

**Comment by GantMan** [comment]
But it doesn't run tests and roll back right?

**Comment by jhnns** [comment]
Does anyone know when yarn is outputting multiple lines? This doesn't happen in our integration tests :(

**Comment by GantMan** [comment]
I just got 3 lines
```
{"type":"warning","data":"../../../../package.json: No license field"}
{"type":"info","data":"Color legend : \n \"<red>\"    : Major Update backward-incompatible updates \n \"<yellow>\" : Minor Update backward-compatible features \n \"<green>\"  : Patch Update backward-compatible bug fixes"}
{"type":"table","data":{"head":["Package","Current","Wanted","Latest","Package Type","URL"],"body":[["danger","3.0.4","3.0.5","3.0.5","devDependencies","https://github.com/danger/danger-js#readme"],["jest","21.2.1","21.2.1","22.1.1","devDependencies","http://facebook.github.io/jest/"]]}}
```

**Comment by jhnns** [comment]
Our tests do already reproduce this issue with the current yarn version. [@GantMan is working on it](https://github.com/peerigon/updtr/pull/61#issuecomment-358128552)

**Comment by GantMan** [comment]
@jhnns - let me know what you think about https://github.com/peerigon/updtr/pull/69 which fixes the issue for Yarn.

**Comment by jhnns** [comment]
Fixed with https://github.com/peerigon/updtr/pull/75, shipped with `v3.0.0` 🚀
